### PR TITLE
docs(hybrid) fix url for cli-arguements of Kong Ingress Controller

### PR DESCRIPTION
**Pull request by pratikthanki**
#### What this PR does / why we need it:

Fix URL in docs which shows the command line arguments that can be passed in Helm for the Kong Ingress Controller

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] PR is based off the current tip of the `main` branch.
- [x] Changes are documented under the "Unreleased" header in CHANGELOG.md
- [ ] New or modified sections of values.yaml are documented in the README.md
- [x] Commits follow the [Kong commit message guidelines](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#commit-message-format)

### CODE REVIEW

File: charts/kong/README.md [hunk 1]
```diff
@@ -712,7 +712,7 @@ section of `values.yaml` file:
 #### The `env` section
 For a complete list of all configuration values you can set in the
 `env` section, please read the Kong Ingress Controller's
-[configuration document](https://github.com/Kong/kubernetes-ingress-controller/blob/main/docs/references/cli-arguments.md).
+[configuration document](https://github.com/Kong/docs.konghq.com/blob/main/src/kubernetes-ingress-controller/references/cli-arguments.md).
```
Comment (pmalek) [inline charts/kong/README.md:715]: I wonder if it wouldn't be better to link to https://docs.konghq.com/kubernetes-ingress-controller/latest/references/cli-arguments/ instead 🤔

Comment (pratikthanki) [reply charts/kong/README.md:715]: I considered that too but the "official docs" are autogenerated from the markdown itself (right?)

Comment (pratikthanki) [reply charts/kong/README.md:715]: That's what I gathered from [Kong/docs.konghq.com](https://github.com/Kong/docs.konghq.com)

Comment (pmalek) [reply charts/kong/README.md:715]: True. Actually, yes, let's keep the reader in this repo (do not link outside). 👍

**Review by pmalek — APPROVED** [review]

**Comment by pmalek** [comment]
Thanks @pratikthanki !

**Pull request merged**